EGI-CLI-Swift-S3.md

```diff
@@ -29,45 +29,44 @@ Last command should return a valid token.
 
 ## Use Openstack
 
+Please configure these environment variables:
+```
+export OS_AUTH_URL=https://identity.cloud.muni.cz/v3
+export OS_AUTH_TYPE=v3oidcaccesstoken
+export OS_PROTOCOL=openid
+export OS_IDENTITY_PROVIDER=egi.eu
+export OS_ACCESS_TOKEN=$OIDC_ACCESS_TOKEN
+export OS_PROJECT_ID=57102d3e06b7476088fe4924370ae170
+export OS_STORAGE_URL=https://object-store.cloud.muni.wift/v1
```

Comment:
Is this the correct object store URL?

Reply:
Good catch, thanks!
Solved in https://github.com/pangeo-data/pangeo-eosc/pull/23/commits/c6f0fb4fd6cb6ffb3e60b6caf327c6ef415ba9aa